pkg/math_utils/utils.go

```diff
@@ -1,23 +1,24 @@
 package math_utils
 
 import (
-	"math/big"
-
 	"github.com/pkg/errors"
+	"math/big"
 )
 
 // Finds a nonnegative integer x < p such that (m * x) % p == n.
 func DivMod(n *big.Int, m *big.Int, p *big.Int) (*big.Int, error) {
-	if m.BitLen() == 0 {
-		return nil, errors.Errorf("m in div_mod(n, m, p) can't be zero")
+	//if m.BitLen() == 0 {
+	//	return nil, errors.Errorf("m in div_mod(n, m, p) can't be zero")
+	//}
```

Comment:
Should we remove this?

Reply:
Thanks for catching that!
Resolved: https://github.com/lambdaclass/cairo-vm_in_go/pull/219/commits/2cb2dd3c17326e0bdc80c2b8d6bbb45595ace988